tests/testthat/test-opencl.R

```diff
@@ -102,3 +102,18 @@ test_that("all methods run with valid opencl_ids", {
   expect_false(is.null(fit$metadata()$opencl_platform_name))
   expect_false(is.null(fit$metadata()$opencl_ids_name))
 })
+
+test_that("error for runtime selection of OpenCL devices if version less than 2.26", {
```

Comment:
I added this test but I'm not sure opencl tests ever get run because I don't see anywhere in the GHA workflows where the env variable "CMDSTANR_OPENCL_TESTS" is set (and the tests only run if that's set). @rok-cesnovar can you remind me, were were unable to get the OPENCL tests to run on GHA? Just trying to remember.

Reply:
We cant run them on GHA unfortunately, as its not supported on these instances.

I run them locally mostly with new releases, but that is mostly it.